### DISCUSSION TITLE
Elementor lightbox Conflict with GTM

### DISCUSSION THREAD

**Discussion by Kaushal-Kava-Projects** [community/community#136437] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

 have issue with Elementor Gallery WordPress Widget, when image clicked, the lightbox not showing correctly otherwise image url open. but if the GTM code removed on the page. The lightbox work fine.

i tried google site plugin , gtm plugin but still this error shows and also i update elementor plugins i expecting from senior developers guide me how to resolve this problem quickly.

**Comment by LiteBrite82** [comment, marked as answer]
Hi @kaushalworld ,

Thanks for being a part of the GitHub Community, we're glad you're here!

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
